docs/src/main/asciidoc/neo4j-guide.adoc

```diff
@@ -362,7 +362,7 @@ public CompletionStage<Response> delete(@PathParam("id") Long id) {
     AsyncSession session = driver.asyncSession();
     return session
         .writeTransactionAsync(tx ->
-            tx.runAsync("DELETE (f:Fruit) WHERE id(f) = $id", Values.parameters("id", id))
+            tx.runAsync("MATCH (a:Fruit) WHERE id(a) = $id DETACH DELETE a", Values.parameters("id", id))
```

Comment:
```suggestion
            tx.runAsync("MATCH (f:Fruit) WHERE id(f) = $id DELETE f", Values.parameters("id", id))
```

Reply:
I'd prefer that we use the same alias as in the other statements.

Your completely right in regards of the `DETACH` modifier, but as we didn't create relations, I'd rather keep it simple. If I remember correctly, I had the statement include the detach, but probably broke it when removing it. Embarrassing.